app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -424,6 +424,12 @@ def advisory_ids(search:)
         ::Katello::Erratum.installable_for_hosts([self]).search_for(search).pluck(:errata_id)
       end
 
+      def filtered_entitlement_quantity_consumed(pool)
+        entitlement = subscription_facet.candlepin_consumer.filter_entitlements(pool.cp_id)&.first
```

Comment:
Is it safe to use `first` here? What if there's more than 1 entitlement returned? I'm not sure how to manually test that scenario, but I see a test with it: https://github.com/Katello/katello/blob/master/test/services/katello/candlepin/consumer_test.rb#L27-L36